### PR TITLE
Create TextArea component

### PR DESCRIPTION
**Pull request by Rheniery**
If applied, the pull request will create the TextArea component.

### Other minor changes:
 - Include placeholder to the Input component examples.
 - Add a PR template to the project.

### Card Link:
N/A

### Design Expected Screenshot
N/A

### Implementation Screenshot or GIF
Seleção_122
Seleção_119
Seleção_120
Seleção_121

### Notes:
N/A

### CODE REVIEW

File: src/components/TextArea/TextArea.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  --textarea-span-line-height: var(--paragraph-m-line-height);
+  --textarea-span-height: var(--height-xs);
+  --textarea-span-color: var(--structure-color-700);
+
+  --textarea-message-negative-color: var(--error-color);
+  --textarea-message-positive-color: var(--success-color);
+}
+
+.textarea {
+  overflow: auto;
```
Comment (diogoribeiro) [inline src/components/TextArea/TextArea.css:54]: could we remove this property? it's better if the textarea's container handles the position

File: src/components/TextArea/TextArea.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+.textarea:focus {
+  outline-color: var(--textarea-outline-color-focus);
+}
+
+.textarea:hover {
+  border: var(--textarea-border-hover, none);
+}
+
+.textarea::placeholder {
```
Comment (diogoribeiro) [inline src/components/TextArea/TextArea.css:80]: does textarea have a placeholder? It seems that it does not

Comment (Rheniery) [reply src/components/TextArea/TextArea.css:80]: Yes, it does.

**Review by alesmit — COMMENTED** [review]
Maybe we could set a min height and "resize: vertical" to only allow vertical resizing (horizontal often breaks layouts)

**Comment by Rheniery** [comment]
> Maybe we could set a min height and "resize: vertical" to only allow vertical resizing (horizontal often breaks layouts)

I think we could not limit it, since it is a package, it's better to let the dev decide how to use it. But yeah, I understand your point, we have to check it with Kaio and Diogo.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+If applied, pull request will ... [Please complete the description]
```
Comment (kaiomagalhaes) [inline .github/pull_request_template.md:1]: *this* pull request will...

File: src/components/Input/Input.stories.tsx [hunk 4]
```diff
@@ -11,17 +11,20 @@ const Template = args => <Input {...args} />;
...
+  size: 'large',
+  placeholder: 'Enter text'
 };
 
 export const PositiveInput = Template.bind({});
 PositiveInput.args = {
   size: 'large',
-  state: 'valid'
+  state: 'valid',
+  placeholder: 'Positive Input'
```
Comment (kaiomagalhaes) [inline src/components/Input/Input.stories.tsx:22]: Let's have the same placeholder "Enter text" in all the inputs

File: src/components/TextArea/TextArea.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+import TextArea from '@/components/TextArea';
+
+export default {
+  title: 'Text Area',
+  component: TextArea
+} as Meta;
+
+const Template = args => <TextArea {...args} />;
+
+export const Default = Template.bind({});
```
Comment (kaiomagalhaes) [inline src/components/TextArea/TextArea.stories.tsx:12]: please add examples with labels

File: src/components/Input/Input.stories.tsx [hunk 6]
```diff
@@ -11,17 +11,20 @@ const Template = args => <Input {...args} />;
 
 export const Default = Template.bind({});
```
Comment (kaiomagalhaes) [inline src/components/Input/Input.stories.tsx:12]: Please add examples with Labels as well. `DefaultWithLabel`, `PositiveWithLabel`, `NegativeWithLabel`

File: src/components/Input/Input.stories.tsx [hunk 7]
```diff
@@ -11,17 +11,20 @@ const Template = args => <Input {...args} />;
...
 Default.args = {
-  size: 'large'
+  size: 'large',
+  placeholder: 'Enter text'
 };
 
 export const PositiveInput = Template.bind({});
```
Comment (kaiomagalhaes) [inline src/components/Input/Input.stories.tsx:18]: Please rename this to "Positive"

File: src/components/Input/Input.stories.tsx [hunk 8]
```diff
@@ -11,17 +11,20 @@ const Template = args => <Input {...args} />;
...
 PositiveInput.args = {
   size: 'large',
-  state: 'valid'
+  state: 'valid',
+  placeholder: 'Positive Input'
 };
 
 export const NegativeInput = Template.bind({});
```
Comment (kaiomagalhaes) [inline src/components/Input/Input.stories.tsx:25]: Please rename this to "Negative"

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

**Review by kaiomagalhaes — APPROVED** [review]

**Review by diogoribeiro — APPROVED** [review]

**Pull request merged**